### PR TITLE
doc: create device sensor help doc.

### PR DESCRIPTION
**Pull request by kkrumm1**
this should help users that get confused when receiving alerts for sensors.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: doc/Support/Device-Sensors.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+LibreNMS has a standard for device sensors they are split into categories.
+This doc is to help users understand device sensors in general, if you need help with developing sensors for a device please see the Contributing + Developing section. 
+
+### Health Sensors
+
+The High and Low values of these sensors can be edited in Web UI by going to the device settings -> Health. There you can set your own custom
+High and Low values. List of these sensors can be found here [Link](../Developing/os/Health-Information.md)
+
+**Note** Some values are defined by the manufactures and other calculated when you add the device into librenms and some are set to null leaving it up
```
Comment (laf) [inline doc/Support/Device-Sensors.md]: None should be null, any in the code that are null will be auto calculated so you only have device defined or calculated ones.

File: doc/Support/Device-Sensors.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+the user define the values, every environment is different and may require user input. 
+
+
+### Wireless Sensors
+
+Some Wireless have  High and Low values of these sensors can be edited in Web UI by going to the device settings -> Health. There you can set your own custom
+High and Low values. List of these sensors can be found here [Link](../Developing/os/Wireless-Sensors.md)
```
Comment (laf) [inline doc/Support/Device-Sensors.md]: I think these auto calculate as well if the devices don't tell us what they are - is that right @murrant

**Comment by kkrumm1** [comment]
@laf good to go?

**Comment by laf** [comment]
Go for it :)

**Pull request merged**